src/source/CppMarshal.scala

```diff
@@ -145,7 +150,7 @@ class CppMarshal(spec: Spec) extends Marshal(spec) {
     }
     def base(m: Meta): String = m match {
       case p: MPrimitive => p.cName
-      case MString => "std::string"
+      case MString => if (spec.cppUseWideStrings) "std::wstring" else "std::string"
```

Comment:
I was able to fix this by merging again with Dropbox/djinni/master. It seems the result of my last merge lost the commit history and squashed the merge down into one commit, I think as a result of me running make clean all that changes several of the generated files.